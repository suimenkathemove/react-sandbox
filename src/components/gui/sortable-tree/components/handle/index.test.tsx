import 'jest-styled-components';

import { cleanup, render } from '@testing-library/react';

import { defaultProps } from './index.stories';

import { Handle } from './';

const setup = () => render(<Handle {...defaultProps} />);

describe('Handle', () => {
  afterEach(() => {
    cleanup();
  });

  test('render', () => {
    const { asFragment } = setup();
    expect(asFragment()).toMatchSnapshot();
  });
});
