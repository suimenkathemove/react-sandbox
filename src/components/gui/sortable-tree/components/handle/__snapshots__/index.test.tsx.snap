// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Handle render 1`] = `
<DocumentFragment>
  .c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 16px;
  height: 24px;
  cursor: -webkit-grab;
  cursor: -moz-grab;
  cursor: grab;
  border-radius: 4px;
}

.c0 svg {
  width: 0.5em;
  height: 0.5em;
  fill: #757575;
}

.c0:hover {
  background-color: rgba(0,0,0,0.05);
}

.c0:hover svg {
  fill: #6f7b88;
}

.c0:active {
  background-color: inherit;
}

.c0:active svg {
  fill: inherit;
}

<button
    class="c0"
  >
    <svg
      viewBox="0 0 20 20"
      width="12"
    >
      <path
        d="M7 2a2 2 0 1 0 .001 4.001A2 2 0 0 0 7 2zm0 6a2 2 0 1 0 .001 4.001A2 2 0 0 0 7 8zm0 6a2 2 0 1 0 .001 4.001A2 2 0 0 0 7 14zm6-8a2 2 0 1 0-.001-4.001A2 2 0 0 0 13 6zm0 2a2 2 0 1 0 .001 4.001A2 2 0 0 0 13 8zm0 6a2 2 0 1 0 .001 4.001A2 2 0 0 0 13 14z"
      />
    </svg>
  </button>
</DocumentFragment>
`;
